SONiC Broadcom build failure in 202211 with error: "TypeError: request() got an unexpected keyword argument 'chunked'"

Hi Team,
I am trying to compile SONiC Broadcom build using 202211 branch code. In this attempt, all compilation process is getting successful but the last stage of sonic-broadcom.bin generation is failing with below indicated error and hence final build is not getting generated.

Please suggest, what is the issue for this failure.

**Error Snippet:**
+ sudo cp files/build_templates/docker_image_ctl.j2 ./fsroot-broadcom/usr/share/sonic/templates/docker_image_ctl.j2
+ **sudo LANG=C DOCKER_HOST= chroot ./fsroot-broadcom /usr/local/bin/generate_shutdown_order.py**
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/docker/api/client.py", line 214, in _retrieve_server_version
    return self.version(api_version=False)["ApiVersion"]
  File "/usr/local/lib/python3.9/dist-packages/docker/api/daemon.py", line 181, in version
    return self._result(self._get(url), json=True)
  File "/usr/local/lib/python3.9/dist-packages/docker/utils/decorators.py", line 46, in inner
    return f(self, *args, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/docker/api/client.py", line 237, in _get
    return self.get(url, **self._set_request_timeout(kwargs))
  File "/usr/local/lib/python3.9/dist-packages/requests/sessions.py", line 600, in get
    return self.request("GET", url, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/requests/sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python3.9/dist-packages/requests/sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/requests/adapters.py", line 486, in send
    resp = conn.urlopen(
  File "/usr/local/lib/python3.9/dist-packages/urllib3/connectionpool.py", line 790, in urlopen
    response = self._make_request(
  File "/usr/local/lib/python3.9/dist-packages/urllib3/connectionpool.py", line 496, in _make_request
    conn.request(
**TypeError: request() got an unexpected keyword argument 'chunked'**

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/generate_shutdown_order.py", line 15, in <module>
    main()
  File "/usr/local/bin/generate_shutdown_order.py", line 8, in main
    manager = PackageManager.get_manager()
  File "/usr/local/lib/python3.9/dist-packages/sonic_package_manager/manager.py", line 1003, in get_manager
    docker_api = DockerApi(docker.from_env(), ProgressManager())
  File "/usr/local/lib/python3.9/dist-packages/docker/client.py", line 96, in from_env
    return cls(
  File "/usr/local/lib/python3.9/dist-packages/docker/client.py", line 45, in __init__
    self.api = APIClient(*args, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/docker/api/client.py", line 197, in __init__
    self._version = self._retrieve_server_version()
  File "/usr/local/lib/python3.9/dist-packages/docker/api/client.py", line 221, in _retrieve_server_version
    raise DockerException(
**docker.errors.DockerException: Error while fetching server API version: request() got an unexpected keyword argument 'chunked'**
+ clean_sys
+ sudo chroot ./fsroot-broadcom umount /sys/fs/cgroup/blkio /sys/fs/cgroup/cpu /sys/fs/cgroup/cpu,cpuacct /sys/fs/cgroup/cpuacct /sys/fs/cgroup/cpuset /sys/fs/cgroup/devices /sys/fs/cgroup/freezer /sys/fs/cgroup/hugetlb /sys/fs/cgroup/memory /sys/fs/cgroup/net_cls /sys/fs/cgroup/net_cls,net_prio /sys/fs/cgroup/net_prio /sys/fs/cgroup/perf_event /sys/fs/cgroup/pids /sys/fs/cgroup/systemd /sys/fs/cgroup /sys
umount: /sys/fs/cgroup/cpu: no mount point specified.
umount: /sys/fs/cgroup/cpu,cpuacct: no mount point specified.
umount: /sys/fs/cgroup/cpuacct: no mount point specified.
umount: /sys/fs/cgroup/net_cls: no mount point specified.
umount: /sys/fs/cgroup/net_cls,net_prio: no mount point specified.
umount: /sys/fs/cgroup/net_prio: no mount point specified.
umount: /sys/fs/cgroup/systemd: no mount point specified.
+ true
+ true
+ sudo LANG=C chroot ./fsroot-broadcom umount /proc
+ true
[  FAIL LOG END  ] [ target/sonic-broadcom.bin ]
make: *** [slave.mk:1214: target/sonic-broadcom.bin] Error 1
Makefile.work:549: recipe for target 'target/sonic-broadcom.bin' failed
make[1]: *** [target/sonic-broadcom.bin] Error 2
make[1]: Leaving directory '/home/sonic/sonic_ci/test/sonic-build-image'
Makefile:39: recipe for target 'target/sonic-broadcom.bin' failed
make: *** [target/sonic-broadcom.bin] Error 2

--- a/build_debian.sh
+++ b/build_debian.sh
@@ -532,7 +532,7 @@ sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install 'setup
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install 'wheel==0.35.1'
 
 # docker Python API package is needed by Ansible docker module as well as some SONiC applications
-sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install 'docker==5.0.3'
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install 'docker==6.1.1'
 
 # Install scapy
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install 'scapy==2.4.4'

